apps/financial-aid/web-osk/src/components/FormLayout/FormLayout.tsx

```diff
@@ -4,11 +4,16 @@ import {
   GridContainer,
   FormStepper,
   Button,
+  Text,
 } from '@island.is/island-ui/core'
 
 import * as styles from './FormLayout.treat'
 
-import { LogoHfj } from '@island.is/financial-aid-web/osk/src/components'
+import {
+  LogoHfj,
+  Login,
+  HasApplied,
+} from '@island.is/financial-aid-web/osk/src/components'
 import { useRouter } from 'next/router'
```

Comment:
useRouter is never used in this file, should we remove it?

We can also remove line 34: `const router = useRouter()` and `setUser` from line 35 and `updateForm` from line 37